src/theme/interaction.rs

```diff
@@ -1,16 +1,22 @@
-use bevy::{ecs::system::SystemId, prelude::*};
-
-use crate::{assets::SfxHandles, audio::sfx::SfxCommands as _};
+use crate::{asset_tracking::LoadResource, audio::SoundEffect, screens::Screen};
+use bevy::{
+    ecs::{system::SystemId, world::Command},
+    prelude::*,
+};
+use std::{collections::VecDeque, marker::PhantomData};
 
 pub(super) fn plugin(app: &mut App) {
     app.register_type::<InteractionPalette>();
+    app.load_resource::<ButtonAssets>();
     app.add_systems(
         Update,
         (
             trigger_on_press,
             apply_interaction_palette,
             trigger_interaction_sfx,
-        ),
+        )
+            .run_if(not(in_state(Screen::Loading)))
+            .run_if(not(in_state(Screen::Splash))),
```

Comment:
Previous code has explicitly not used chained `run_if`s as that makes it unclear whether we are AND or ORing. I *personally* like chaining them, but I want the code to be consistent and I definitely see the argument here.

Reply:
I personally think this reads better than the `and_then` chain, but I don't have strong opinions here.

Reply:
@benfrankel using De Morgan, both 
```rust
.run_if(not(in_state(Screen::Loading).and_then(in_state(Screen::Splash)))
```
and 
```rust
.run_if(not(in_state(Screen::Loading)).and_then(not(in_state(Screen::Splash)))
```
seem very unreadable to me. I'm tending heavily towards keeping it as in shown in the PR. Would that be alright for you?

Reply:
That first one should be a `.or_else`, I believe.

When I read "run if A, run if B", it sounds to me like "if A is true, run the system; if B is true, run the system", which would be OR. But the actual behavior is surprisingly "if A is NOT true, DO NOT run the system; if B is NOT true, DO NOT run the system", which is AND. That only makes sense if you already know that that's what will happen. And `and_then` exists to be used, presumably.

If the `.run_if` x2 remains, it needs a comment at least.

Reply:
@benfrankel thanks, fixed.
I agree, a comment should at least be there, even though it is "standard Bevy behavior". The syntax is counterintuitive when you read it the first time.

Reply:
To expand on my comment about unreadableness: IME chained `run_if`s will be formatted more nicely by cargo fmt than `and_then`. If it is formatted nicely in this case, I have no criticism.

Reply:
I've opted for `and_then`